### PR TITLE
wasabi

### PR DESCRIPTION
**Pull request by izaakf**

### CODE REVIEW

File: task-definition.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "ipcMode": null,
-  "executionRoleArn": "arn:aws:iam::143335205145:role/ecsTaskExecutionRole",
+  "executionRoleArn": "monkaS",
```
Comment (sentriz) [inline task-definition.json:3]: nit:
```suggestion
  "executionRoleArn": "pogcity",
```

**Comment by sentriz** [comment]
other than that, LGTM :tada:

**Comment by sentriz** [comment]
thanks :D

**Comment by sentriz** [comment]
updating... https://github.com/sentriz/wazaap/runs/895909673?check_suite_focus=true

**Comment by izaakf** [comment]
> updating... https://github.com/sentriz/wazaap/runs/895909673?check_suite_focus=true

Looking good!!!

**Pull request merged**